boards/nxp/s32k148_evb/s32k148_evb.dts

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include <zephyr/dt-bindings/pwm/pwm.h>
+#include <arm/nxp/nxp_s32k148.dtsi>
+#include "s32k148_evb-pinctrl.dtsi"
+
+/ {
+	model = "NXP S32K148EVB";
+	compatible = "nxp,s32k148_evb";
+
+	chosen {
+		zephyr,sram = &sram_l;
+		zephyr,flash = &flash0;
+		zephyr,console = &lpuart1;
+		zephyr,shell-uart = &lpuart1;
+		zephyr,uart-pipe = &lpuart1;
+		zephyr,canbus = &flexcan0;
+	};
+
+	aliases {
+		led0 = &led1_red;
+		led1 = &led1_green;
+		led2 = &led1_blue;
+		pwm-led0 = &led1_red_pwm;
+		pwm-led1 = &led1_green_pwm;
+		pwm-led2 = &led1_blue_pwm;
+		red-pwm-led = &led1_red_pwm;
+		green-pwm-led = &led1_green_pwm;
+		blue-pwm-led = &led1_blue_pwm;
+		sw0 = &button_3;
+		sw1 = &button_4;
+		i2c-0 = &lpi2c0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led1_red: led_0 {
+			gpios = <&gpioe 21 GPIO_ACTIVE_HIGH>;
+			label = "LED1_RGB_RED";
+		};
+
+		led1_green: led_1 {
+			gpios = <&gpioe 22 GPIO_ACTIVE_HIGH>;
+			label = "LED1_RGB_GREEN";
+		};
+
+		led1_blue: led_2 {
+			gpios = <&gpioe 23 GPIO_ACTIVE_HIGH>;
+			label = "LED1_RGB_BLUE";
+		};
```

Comment:
I think you want `ACTIVE_LOW` for these LEDs?

![image](https://github.com/user-attachments/assets/d52648d0-e6f8-4680-86a6-a84c6c9b63d1)